source/contributing/how-to-migrate.md

```diff
@@ -0,0 +1,34 @@
+# How to Migrate Documentation - Step-by-Step Process
+
+Migrating documentation is often crucial when reorganizing any project. As such, below is a list of instructions and guidelines to aid you when embarking on the migration journey.
+
+1. **Licensing**
+   1. Familiarize yourself with the licenses governing the documentation you intend to migrate.
+   2. Verify if the license of the documentation is compatible with this project's current license.
+   3. If the licenses align, proceed with the migration. Otherwise, follow the steps below.

```

Comment:
Suggestion: Make it clear that "the steps below" refer to steps 4-7 and not the rest of the document below this bullet.